targets/wasm_exec.js

```diff
@@ -313,6 +313,26 @@
 					//"syscall/js.valueInstanceOf": (sp) => {
 					//	mem().setUint8(sp + 24, loadValue(sp + 8) instanceof loadValue(sp + 16));
 					//},
+
+					// copyBytesToJS(dst ref, src []byte) (int, bool)
+					// Originally copied from upstream Go project, then modified:
+					//   https://github.com/golang/go/blob/3f995c3f3b43033013013e6c7ccc93a9b1411ca9/misc/wasm/wasm_exec.js#L404-L416
+					// param4 and param5 (below) are likely "length" variables of some sort
+					"syscall/js.copyBytesToJS": (ret_addr, dest_addr, source_addr, param4, param5) => {
```

Comment:
The calling convention changes have changed this function conceptually to the following:

```go
func copyBytesToJS(returnValue *struct{int; bool}, dst *ref, srcPtr *byte, srcLen, srcCap uintptr)
```

The following two rules apply to it:

  * large return values are passed as a pointer-to-struct and prepended to the parameter list
  * `int64`/`uint64` (the `ref` type) are passed as a pointer-to-value
  * slices are split into 3 parts: the pointer to the underlying array, the length, and the capacity.

Reply:
Thanks.  I'll update things accordingly over the next few days. :smile:

Reply:
Any chance you can take a look at this PR? The next release is getting close.

Reply:
Yep, I can make time this weekend. :smile: